apps/librelingo_json_export/librelingo_json_export/challenge_types.py

```diff
@@ -3,6 +3,8 @@
 from librelingo_utils import get_dumb_opaque_id, audio_id, clean_word, iterate_phrases
 from .dictionary import _define_words_in_sentence
 
+def _remove_special_characters_for_display(string):
+    return string.replace('}', '').replace('{', '')
```

Comment:
same as here: https://github.com/kantord/LibreLingo/pull/1452/files#r671888194

and while at it, I would probably rename it to something more generic, like `_format_text_for_display`, or whatever that's better that you can think of. The reason why I say this is because it's possible that soon new special characters will have to be removed, so it will require less work to introduce those if the function name doesn't have to be changed